### DISCUSSION TITLE
Pomash 大体完工

### DISCUSSION THREAD

**Discussion by giscus** [JmPotato/Pomash#20] [Announcements]
# Pomash 大体完工

https://ipotato.me/article/6

**Comment by feiandxs** [comment]
本来心想很勇猛，虽然我也喜欢前后端一体，但想到写复杂的页面还是很令人发怵，远远没有前后端分离愉快，甚至用屎一样的 nextjs 也会体验好一点。 但看了下居然是把 react 直接引入进来了……

但我还是觉得虽然这样很大程度上能能提升前端编写的体验，但似乎这个思维逻辑还是得走的有点绕，不够前后端完全分开那样清爽。好像有点无解。

都怪前端发展成这个样子。
